en/docs/testing/unit-testing-in-build-pipeline.md

```diff
@@ -0,0 +1,28 @@
+# Unit Testing in Choreo Build Pipeline
+
+Choreo supports unit testing as part of the build pipeline, enabling users to validate their applications before deployment. This feature ensures that code changes are tested early, reducing the risk of defects in production.
+
+!!! note
+        Currently, Choreo only supports unit testing for the `WSO2 MI` buildpack.
```

Comment:
Currently, Choreo supports unit tests only for the **WSO2 MI** buildpack.

Reply:
We would need to mention the component types too yeah? As of now this is supported only for Schedules Task, Manual Task, and Event Handler component types is it?

Reply:
This supports any MI component type regardless of the type